ladybug_comfort/hi.py

```diff
@@ -94,3 +94,44 @@ def heat_index_warning_category(hi):
         category = 4
 
     return category
+
```

Comment:
I see that you added `humiex` to the `hi` module but I think I would prefer to have it in a separate .py file labeled `humidex` unless these two metrics share some history or relation to one another. Did you uncover any relation between the two as you were putting together the docstring?

Reply:
The primary reason I added them to the `hi.py` module was that the Heat Index and Humidex functions were in the same module in the legacy code.

On further research, it seems they are both measures of what the heat actually "feels" like based on air temp and humidity, however the Humidex is used in Canada and is unitless (though often reported as degrees C) whereas in the United States, they measure relative temperature with the Heat Index (in degrees Farenheit).

I do know that they calculations differ in result and can't be compared based on [this info from the The Weather Network](https://www.theweathernetwork.com/news/articles/humidex-vs-heat-index-whats-the-difference/10243)

That being said, I moved the Humidex functions with https://github.com/ladybug-tools/ladybug-comfort/pull/41/commits/0099bfd8dba3e321cd707c6df52a5021ed19d2a5
I don't know if they are similar enough to share a module.

Reply:
Thanks for the thorough research and I agree with the conclusion that you came to (separate modules is really the best). FYI the reason why the two models were originally in the same.py  file is because our older coding practices were not good. In fact, there are 18 different comfort models in that one file and so it will be very nice to have the comfort models organized here in separate modules.